xbmc/music/tags/MusicInfoTag.cpp

```diff
@@ -381,10 +381,10 @@ void CMusicInfoTag::SetAlbumArtist(const std::string& strAlbumArtist)
   }
 }
 
-void CMusicInfoTag::SetAlbumArtist(const std::vector<std::string>& albumArtists)
+void CMusicInfoTag::SetAlbumArtist(const std::vector<std::string>& albumArtists, bool FillDesc /* = false*/)
 {
   m_albumArtist = albumArtists;
-  if (m_strAlbumArtistDesc.empty()) 
+  if (m_strAlbumArtistDesc.empty() || FillDesc) 
```

Comment:
I am avoiding having to change the many places setartist is called. I would not have designed it that way from scratch, but that boat sailed long ago. For Jarvis I really think it is the best we can do.

I got some test builds up 45f831, I am hoping that Scott will do some testing for us.